maths-club-pack/content/facilitator/domino-tilings.md

```diff
@@ -0,0 +1,54 @@
+# Domino Tilings
+
+## Introduction
+The idea of this exercise is to explore the possibility or otherwise of tiling (covering) certain shapes by dominos. While the question appears geometric, there the combinatorics of the situation also turns out to be important. One idea that learners should get from this exercise is that sometimes a construction is possible while other times it will not be; good students should also start to be able to make arguments (proofs!) for why a certain tiling is not possible. The proof of the most advanced example introduces a basic
+idea in combinatorial arguments, that or colouring.  
+
+ Experimentation can be helped by the availability of squared paper, or paper and scissors to make the dominos!
+
+
+## Solution    
+
+a. tilings clearly exists (this could be done at the beginning together so that students get the idea).Experimentation will suggest that in part  
+b. there is no solution, and good students will also spot the fact that the reason is that there is an odd
+number of squares. Indeed, removing one square as in part  
+c. will give a shape that can be tiled again. For part
+d. experimentation quickly gives a tiling; for part  
+e., it can be shown by trying essentially all possibilities that there is no solution again.  
+
+
+How so, since there is an even number of squares in both cases, and we removed two corners in each
+case? Well, the trick is to think of a chessboard colouring of our shape (it should be suggested to students
+to think of a chessboard, or explain to them what that looks like, but initially without further comment).
+The point is that no matter how one puts down a domino, it will always cover one black and one white
+square. So in part e. no tiling can exist since the coloured version has a different number of black and white
+squares:  
+
+![](../../images/domino-tilings-4.png)  
```

Comment:
I think this image got cropped at the left a little bit. I'm not sure if you are taking screenshots/clips yourself from the pdf, or if mike shared the word document versions in which case you can just right-click on any image and say `Save image as`, which might be easier.